### DISCUSSION TITLE
CH32V30x: Qingke-V4F现在支持了多少功能？

### DISCUSSION THREAD

**Discussion by syx-413** [ch32-rs/ch32-hal#42] [Q&A]
Supported Devices and Peripherals 表格中没有V3 以上的支持，但似乎能够操作blink led

**Comment by andelf** [comment]
Hi, CH32V3 和 CH32V2 是等价的, 区别只有 FPU 和部分外设. 代码可以通用. 
方便的话可以提PR修改下 "Supported Devices and Peripherals"
